strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/AccountControllerTest.java

```diff
@@ -35,7 +35,7 @@
  * @author Pablo Tirado
  */
 @IntegrationTest
-@ActiveProfiles(profiles = "test")
+@ExtendWith(SpringExtension.class)
```

Comment:
@carlspring @fuss86  why we need it here?

there is already `@SpringBootTest` in `@IntegrationTest`

Reply:
Thanks for pointing it out! At the time, we weren't yet using `@SpringBootTest`. We will have to fix this.